src/core_modules/capture-core/components/WidgetEventEdit/WidgetEventEdit.container.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React, { type ComponentType } from 'react';
+import { withStyles } from '@material-ui/core';
+import { spacersNum } from '@dhis2/ui';
+import { Widget } from '../Widget';
+import type { Props } from './widgetEventEdit.types';
+import { EditEventDataEntry } from '../Pages/ViewEvent/EventDetailsSection/EditEventDataEntry/';
+import { ViewEventDataEntry } from '../Pages/ViewEvent/EventDetailsSection/ViewEventDataEntry/';
+import { pageMode } from '../Pages/EnrollmentEvent/EnrollmentEventPage.const';
+import { NonBundledDhis2Icon } from '../NonBundledDhis2Icon';
+
+const styles = {
+    header: {
+        display: 'flex',
+        alignItems: 'center',
+    },
+    name: {
+        margin: `0 ${spacersNum.dp4}px`,
+    },
+};
+
+export const WidgetEventEditPlain = ({
+    classes,
+    programStage,
+    programStage: { name, icon },
+    mode,
+}: Props) => (
+    <div data-test="widget-enrollment">
+        <Widget
+            header={
+                <div className={classes.header}>
+                    <NonBundledDhis2Icon
```

Comment:
Maybe left-align the program stage name if there is no icon object at all. See `StageOverview.component.js`.

Reply:
Thanks for the feedback! I added it.